components/admin/requests/processing/TasksCard.tsx

```diff
@@ -89,7 +91,7 @@ export default function ProcessingTasksCard({ applicationId }: ProcessingTasksCa
     await mailOut({ variables: { input: { applicationId, appMailed } } });
     refetch();
   };
-
+  const [reviewedRequestInformation, setReviewedRequest] = useState<boolean>(false);
```

Comment:
```suggestion
  const [hasReviewedRequest, setHasReviewedRequest] = useState<boolean>(false);
```
This makes it more clear what the purpose of the state variable is

Reply:
Removed this state, it was just to show Chris a working modal. I think he is setting up the states differently for his ticket, so this will be removed anyways.